src/NUnitEngine/Addins/nunit-project-loader/NUnitProject.cs

```diff
@@ -156,7 +156,7 @@ public string GetSetting(string name, string defaultValue)
         /// </summary>
         internal XmlNode RootNode
         {
-            get { return xmlDoc.FirstChild; }
+            get { return xmlDoc.DocumentElement ; }
```

Comment:
Good pickup. There are other places where we use FirstChild and should probably use DocumentElement.